src/server/multi_test.cc

```diff
@@ -531,9 +533,9 @@ TEST_F(MultiTest, Watch) {
   ASSERT_THAT(Run({"exec"}), kExecSuccess);
 }
 
-TEST_F(MultiTest, MultiOOO) {
-  GTEST_SKIP() << "Command squashing breaks stats";
-
+// If all commands fit into a single batch, it can be run as a single hop, without separate hops for
+// locking and unlocking
+TEST_F(MultiTest, MultiSingleHop) {
   auto fb0 = pp_->at(0)->LaunchFiber([&] {
     for (unsigned i = 0; i < 100; i++) {
       Run({"multi"});
```

Comment:
I would add another call to Run({"rpush", "a", "bar"}); inside the multi transaction here and check after execution size of a to check that both commands are executed